transformation_test/transformation_test.go

```diff
@@ -204,9 +202,19 @@ func generateFluentBitConfigs(ctx context.Context, transformationTest transforma
 	output := fluentbit.Component{
 		Kind: "OUTPUT",
 		Config: map[string]string{
-			"Name":   "stdout",
-			"Match":  "*",
-			"Format": "json",
+			"Match":                         "*",
+			"Name":                          "stackdriver",
+			"Retry_Limit":                   "3",
+			"http_request_key":              "logging.googleapis.com/httpRequest",
+			"net.connect_timeout_log_error": "False",
+			"resource":                      "gce_instance",
```

Comment:
Is fluent-bit going to try to do resource detection (and thus make this test fail on a cloudtop because it adds resource labels)?

Reply:
Tested it out with no issues.